botocore/loaders.py

```diff
@@ -340,7 +340,7 @@ def list_api_versions(self, service_name, type_name):
         return sorted(known_api_versions)
 
     @instance_cache
-    def load_service_model(self, service_name, type_name, api_version=None):
+    def load_service_model(self, service_name, type_name):
```

Comment:
I'm wondering for now if it make sense to hold off on updating the `loaders` module for removing `api_version` right now? Given that we have not really addressed the structure of the data directory yet, it seems reasonable to keep the data loader interface in line with how the models are organized on disk.